src/content/docs/style-guide/processes-procedures/embed-images.mdx

```diff
@@ -35,21 +35,33 @@ Here are some things to keep in mind when you're creating an image:
 
 ## Add an image [#add]
 
-Our docs site images are stored in the `images` directory. They're organized by navigation section name.
+Our doc site images are stored in individual `images` directories at the root level of each taxonomy category. These `images` directories contain all the images used in the docs for that category. 
 
-For example, the images for the Build an App Guide are located in the build-an-app directory.
+To add an image from scratch:
+1. Find the images directory for your doc. For example, if your docs lives in `docs/style-guide/processes-procdures` you would use the `docs/style-guide/processes-procdures/images` directory. If the `images` directory does not exist, create one in [VScode](https://code.visualstudio.com/docs/getstarted/tips-and-tricks#_create-or-open-a-file) or Finder.
+2. Place your image in the `images` directory. Give the image a descriptive file name: `fso-ui-overview.png` is much better than `123-go-dawgs.png`. 
+3. Follow the [steps below](#embed) to embed the image in a doc. 
 
 ## Embed an image [#embed]
 
-You can use Markdown to embed an image:
+Use markdown to embed an image in a doc. The basic structure:
 
 ```
 ![alt text](<var>PATH_TO_IMAGE</var> "Image title text")`
 ```
 
+Here's a filled in example:
+
+```
+![An image showing an overview of the synthetics UI](./images/synthetics-ui-overview.png "Synthetics UI overview")`
+```
+
 ## Update an image [#update]
 
-To update an image, find its location, and then upload a new image to GitHub with the same filename.
+To update an image:
+1. Delete the original image file in the corresponding `images` directory.
+2. Place the new image file in the same `images` directory.
+3. Ensure the name image file has the same name as the original file. 
```

Comment:
Looks like you want to reword this step 3. I think you could just delete that first "name" there so it says: "Ensure the image file has the..."